lib/membrane_http_adaptive_stream/hls.ex

```diff
@@ -40,6 +41,29 @@ defmodule Membrane.HTTPAdaptiveStream.HLS do
         "#EXT-X-MAP:URI=#{header_name}"
       ]
     end
+
+    @impl true
+    def serialize({:creation_time, date_time}) do
```

Comment:
what is it for?

Reply:
The `#EXT-X-PROGRAM-DATE-TIME` allows a player to estimate latency relative to the moment a segment has been produced. Quite helpful for debugging and client-side metrics collection.